### DISCUSSION TITLE
Show all installed plugins in the "Craft 4 Upgrade" utility

### DISCUSSION THREAD

**Discussion by aelvan** [craftcms/cms#11101] [Ideas] [answered]
Currently the "Craft 4 Upgrade" utility only show plugins that are in the plugin store. It would be nice if it could also _list_ installed plugins that are not in the store, on the same page. I understand that it can't easily check if there's a 4.0 compatible version, but it would be nice to be able to see which custom/internal plugins we need to manually check.

**Comment by brandonkelly** [comment, marked as answer]
Good call. Just added any uninstalled/unknown plugins to the mix for the next release.

**Reply by brandonkelly** [reply]
3.7.41 is out now with that change.

**Reply by aelvan** [reply]
Looks great!
